include/flox/core/exceptions.hh

```diff
@@ -25,10 +25,34 @@ enum error_category {
   EC_OKAY = 0,
   // Returned for any exception that doesn't have an error_code(), i.e.
   // exceptions we haven't wrapped in a custom exception
-  EC_FAILURE               = 1,
-  EC_FLOX_EXCEPTION        = 100,
-  EC_PKG_QUERY_INVALID_ARG = 101,
-  EC_TOML_TO_JSON          = 102
+  EC_FAILURE                     = 1,
+  EC_FLOX_EXCEPTION              = 100,
```

Comment:
I'm not sure if we actually ever want to throw a FloxException - we could make it an abstract base class instead